hot-reload.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: page
+title: Hot Reloading Flutter Apps
+permalink: /hot-reload/
+---
+
+* TOC
+{:toc}
+
+## Using hot reload
+
+Flutter's hot reload feature helps you quickly and easily experiment, build
+UIs, add features and fix bugs. Hot reload works by injecting updated source
+code files into the running Dart Virtual Machine (VM). After the VM updates
+classes with the new versions of fields and functions, the Flutter framework
+automatically rebuilds the widget tree, allowing you to quickly view the effects
+of your changes.
+
+To hot reload a Flutter app:
+
+1.  Run the app from a supported IntelliJ IDE or a terminal window. Either a
+physical or virtual device can be the target.
+1.  Modify one of the Dart files in your project. Most types of code changes can
+be hot reloaded; for a list of changes that require a full restart, see
+[Understanding limitations](#understanding-limitations).
+1.  If you're working in an IntelliJ IDE that supports Flutter's IDE tools,
+click the Hot Reload button on the toolbar:
```

Comment:
Hot reloading on save is now the default; see https://flutter.io/intellij-ide/#fast-edit-and-refresh-development-cycle

Reply:
Added "either... or select *File > Save All**.